.gitpod.yml

```diff
@@ -0,0 +1,20 @@
+tasks:
+  - before: |
+      sudo apt update -y
+      DEBIAN_FRONTEND=noninteractive sudo apt-get install -y bison python3-dev libcppunit-dev build-essential cmake autoconf tcl pkg-config git curl patch libtool-bin
+
+      # install bazel
+      sudo apt install -y apt-transport-https curl gnupg
+      curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg
```

Comment:
we use bazelisk for zetasql.
have a look in https://github.com/4paradigm/zetasql/blob/feat/hybridse-zetasql/.github/workflows/main.yml#L87

Reply:
this follows the [Download Pre-Compiled Thirdparty](https://openmldb.ai/docs/en/main/deploy/compile.html#:~:text=Method%20Two%3A%20Download%20Pre%2DCompiled%20Thirdparty). is bazel/bazelisk necessary in this method?

Reply:
not necessary, bazel only required if you want to compile thirdparty (zetasql) from source